src/common/type/stringList.h

```diff
@@ -95,6 +95,18 @@ strLstExists(const StringList *const this, const String *const string)
     return lstExists((List *)this, &string);
 }
 
+// Find string index in the list
+typedef struct StrLstFindIdxParam
+{
+    VAR_PARAM_HEADER;
+    bool required;
+} StrLstFindIdxParam;
+
+#define strLstFindIdxP(this, string, ...)                                                                                            \
```

Comment:
Define continuation '\' is at column 134 vs 132

Reply:
Good catch, fixed.